app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt

```diff
@@ -233,13 +225,11 @@ class TypingGameActivity : GameActivity() {
      * Function to set scores in the end of the game
      */
     private fun setScores() {
-        if(::gameManager.isInitialized && FireBaseAuthenticator.isLoggedIn()){
-            dataGetter.updateFieldInt(FireBaseAuthenticator.getCurrentUID(), "totalGames", 1, method = "sum")
-            dataGetter.updateFieldInt(FireBaseAuthenticator.getCurrentUID(), "correctSongs", gameManager.getCorrectSongs().size, method = "sum")
-            dataGetter.updateFieldInt(FireBaseAuthenticator.getCurrentUID(), "bestScore", gameManager.getScore(), method = "best")
-            dataGetter.updateSubFieldInt(FireBaseAuthenticator.getCurrentUID(), gameManager.getScore(), "scores", gameManager.gameMode, method = "best")
+        if (::gameManager.isInitialized) {
+            super.setScores(gameManager)
         }
     }
+
     /**
```

Comment:
```suggestion
    /*
```
same as above, Javadoc is for a single function

Reply:
Maybe adding a javadoc for each could be great as their part of public interface, even if they are only for testing purpose.

Reply:
this is a nice comment, but not the focus of my PR. I think leaving the task to the people who do refactoring will be better.

Reply:
okay will do myself the change so. We will forget it otherwise